src/xdocs/config_imports.xml

```diff
@@ -818,6 +822,42 @@ import android.*;
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the closest to the current package described inside import control file.</li>
+            <li>If there is matching allow/disallow rule inside the current package then check returns "allowed" or "disallowed" message.</li>
+            <li>If there is no matching allow/disallow rule inside the current package then checks goes up to the parent and continues checking.</li>
+            <li>If there is no matching allow/disallow rule inside the root level then import is disallowed by default.</li>
```

Comment:
How about:
`If there is no matching allow/disallow rule in any of the sub-packages, including the root level, then the import is disallowed by default.`